server/src/services/database.service.spec.ts

```diff
@@ -23,30 +25,18 @@ describe(DatabaseService.name, () => {
     [{ vectorExt: DatabaseExtension.VECTORS, extName: 'pgvecto.rs', minVersion: new Version(0, 1, 1) }],
     [{ vectorExt: DatabaseExtension.VECTOR, extName: 'pgvector', minVersion: new Version(0, 5, 0) }],
   ] as const)('init', ({ vectorExt, extName, minVersion }) => {
-    let fatalLog: MockInstance;
-    let errorLog: MockInstance;
-    let warnLog: MockInstance;
-
     beforeEach(() => {
-      fatalLog = vitest.spyOn(ImmichLogger.prototype, 'fatal');
-      errorLog = vitest.spyOn(ImmichLogger.prototype, 'error');
-      warnLog = vitest.spyOn(ImmichLogger.prototype, 'warn');
```

Comment:
these tests are so much nicer now that the logger in dependency injected instead of using a spy